schema/node.json

```diff
@@ -8,7 +8,8 @@
   "properties": {
     "@id": {
       "type": "string",
-      "format": "uri-reference"
+      "format": "uri-reference",
+      "x-go-name": "HyperionId"
     },
     "@type": {
```

Comment:
We should do this for all Hyperion properties. I imagine that `type` and `links` are very popular names for properties.